### ISSUE TITLE
Play button doesn't work when playback has finished

### ISSUE THREAD

**Issue by mfat** [Rafostar/clapper#252]
After the playback has finished, play button doesn't work anymore.
The expected behavior is restarting playback from the beginning when the play button is clicked.

**Comment by medeotl** [comment]
This is particular annoying in this use case:
- browse you hard drive folders to get rid of usefulness video files
- get some clips (short files of few seconds/minutes)
- expected behaviour: you can easily re-watch them to make some decision
- current behaviour: you have to close program and click on the file again

**Comment by Rafostar** [comment]
> current behaviour: you have to close program and click on the file again

You can also click on elapsed time button and select the same file from the playlist there to play it again, but yes I agree that hitting spacebar would be even easier and faster.

Some time ago this topic was also brought up on Clapper matrix channel. The current plan/conclusion is to stop resetting the UI, since there is no benefit in doing so. Make progress bar simply remain fully filled, so you can easily rewind. And to make play button and spacebar restart playback from beginning in such state.

**Comment by nekohayo** [comment]
Let's also allow seeking directly back to any position even with the pipeline paused, without necessarily doing playback.

In terms of UI:

For convenience, instead of an option to "do nothing" vs "freeze frame" vs "close the app" (are there really people who use this, particularly the 3rd one?), I would suggest that Clapper only does this, by default: when playback ends, show the last frame of the video but gaussian-blurred and 50% brightness (or 25%?), with an overlaid text and button that say:

> ## Title of the video without file extension
> 
> Press `Spacebar` or use the button below to restart playback from the beginning.
>
> [ Restart playback ]

**Comment by Rafostar** [comment]
> For convenience, instead of an option to "do nothing" vs "freeze frame" vs "close the app"

Yes. We should simplify this to "freeze frame" only (with option to change this removed). This will allow to queue another item for playback easily when finished without closing the app.
